python/requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest
 cloudpickle
 numpy>=1.10.0
 six
+futures
```

Comment:
What does this do when running under Python 3?

Reply:
It's basically a no-op because `concurrent.futures` is part of the py3 standard library

Reply:
Cool, just curious.

Reply:
I was wrong, this should not be installed in py3. Sorting out a fix